pkg/config/ebpf_tracer.go

```diff
@@ -36,15 +37,19 @@ type EBPFTracer struct {
 	// before sending a wakeup request.
 	// High values of WakeupLen could add a noticeable metric delay in services with low
 	// requests/second.
+	// Must be at least 0
 	// TODO: see if there is a way to force eBPF to wakeup userspace on timeout
 	WakeupLen int `yaml:"wakeup_len" env:"OTEL_EBPF_BPF_WAKEUP_LEN"`
 
 	// BatchLength allows specifying how many traces will be batched at the initial
 	// stage before being forwarded to the next stage
+	// Must be at least 1
 	BatchLength int `yaml:"batch_length" env:"OTEL_EBPF_BPF_BATCH_LENGTH"`
 
 	// BatchTimeout specifies the timeout to forward the data batch if it didn't
 	// reach the BatchLength size
+	// TODO (pino): do we need to decide minimum and maximum values? Ask Nimrod and Mattia
```

Comment:
I'd not enforce every single config option, the important thing is that there are good defaults

Reply:
So for BatchTimeout you think that we can just check if it's not <=0?